File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    
    const nums1 = {}
    
    for (let num of nums){
        if (nums1[num] == true) return true
        else nums1[num] = true;
    }
    
    return false

};

File: 0217-contains-duplicate/0217-contains-duplicate.java
import java.util.HashMap;
import java.util.Map;


class Solution {
    public boolean containsDuplicate(int[] nums) {
        // Create a Map using HashMap
        Map<Integer, Boolean> map = new HashMap<>();
      
       //here, we can write all other required operations
        for (int i = 0; i < nums.length; i++){
            if (map.get(nums[i]) == null){
                map.put(nums[i], true);
            }else return true;
        }

        return false;
    }
}